.gitignore

```diff
@@ -27,3 +27,4 @@ npm-debug.log
 /config/*.secret.exs
 .env
 cover
+*.pem
```

Comment:
I'm not sure if this line is still necessary :thinking: 
I think it's useful if user generate a private key at the root of the application :+1: